routing-configurer/src/main/java/no/vegvesen/ixn/federation/qpid/QpidClient.java

```diff
@@ -28,11 +28,11 @@
 
 
 	private final Logger logger = LoggerFactory.getLogger(QpidClient.class);
-	private static final String EXCHANGE_URL_PATTERN = "%s/api/latest/exchange/default/%s";
-	private static final String QUEUES_URL_PATTERN = "%s/api/latest/queue/default/%s";
-	private static final String PING_URL_PATTERN = "%s/api/latest/virtualhost/default/%s";
+	private static final String EXCHANGE_URL_PATTERN = "%s/api/latest/exchange/%s/%s";
```

Comment:
Can we separate the settings for virtualhostnode-name and virtualhost name? Originally the virtualhostnode-name was "default".